services/compute/availabilityset/wssd.go

```diff
@@ -131,6 +131,7 @@ func (c *client) getAvailabilitySetRequest(opType wssdcloudcommon.Operation, gro
 		AvailabilitySets: []*wssdcloudcompute.AvailabilitySet{},
 	}
 
+	return nil, errors.Wrapf(errors.InvalidGroup, "Group specified Group[%s]", group)
```

Comment:
Looks like local test changes. Please revert.

Reply:
removed these changes. they were added just for my local testing